databases/core.py

```diff
@@ -411,13 +416,26 @@ def driver(self) -> str:
             return ""
         return self.components.scheme.split("+", 1)[1]
 
+    @property
+    def userinfo(self) -> typing.Optional[bytes]:
+        if self.components.username:
+            info = self.components.username
+            if self.components.password:
+                info += ":" + self.components.password
+            return info.encode("ascii")
```

Comment:
Some DBs, for example, Postgres, allow using any crazy Unicode user names and passwords.

Reply:
can change this to "utf-8", i chose `ascii` because it was how it is encoded into bytes in `httpx`